examples/threadExample/src/main/java/io/realm/examples/threads/ThreadFragment.java

```diff
@@ -161,8 +158,8 @@ public void onPause() {
     }
 
     @Override
-    public void onDestroy() {
-        super.onDestroy();
+    public void onStop() {
+        super.onStop();
```

Comment:
using `onCreate`/`onDestroy` for **Fragments** is not reliable (mostly because `onDestroy` is not guarantee to be called (see my post on HelpScout)  it's fine for Activities though